.travis.yml

```diff
@@ -51,7 +51,7 @@ before_script: # TODO add golangci yaml config
 script:
   - make test
   - if [ "$TRAVIS_CPU_ARCH" == "amd64" ]; then
-      make all.amd64;
+      make all.amd64 || exit 1;
```

Comment:
we can use `travis_terminate 1;`

Reply:
No documentation found regarding that. 
Found the `travis_terminate 1` thing in a deprecated api.
I asked the same in the Travis community.
https://travis-ci.community/t/how-to-stop-the-execution-of-travis-if-script-exits-with-an-error/8556